src/index.html

```diff
@@ -1,13 +1,75 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h1 class="product-card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h1>
+
+      <h2 class="product-card__code">
+        Код товара: 195434
+      </h2>
+
+      <div class="rating product-card__rating">
```

Comment:
Nice work! I even reread a little theory again because of a review:)
So, "rating" is without prefix and it is block, you need to make separate rating.css file for it and you don`t use product-card__rating, as I see

I will recomend maybe to leave "product-card__rating and move all styles from "rating" to it and no need to create separate file. But as you wish.